civihr_employee_portal/css/custom.css

```diff
@@ -159,3 +159,72 @@ span.appraisals-employee-legend {
   vertical-align: middle;
   padding-left: 10px;
 }
+
+/* styles for my leave report */
+body{
+    font-family: 'Lato', sans-serif;
```

Comment:
Why are you defining a `body` rule here in the middle of this file? Isn't the `font-family` defined somewhere else already?